drivers/azure/azureutil/azureutil.go

```diff
@@ -22,7 +22,10 @@ import (
 )
 
 const (
-	storageAccountPrefix = "vhds" // do not contaminate to user's existing storage accounts
+	// This key indicates that the resource specified is managed by rancher
+	ManagedByRancherKey = "managed-by-rancher-tagged-by-machine-do-not-remove"
```

Comment:
This is pretty verbose, can we reduce it to just `rancher-managed`?

Reply:
Sure; will be updated in the next push